tests/MatrixDotNet.PerformanceTesting/Other/BenchCount.cs

```diff
@@ -5,7 +5,7 @@
 namespace MatrixDotNet.PerformanceTesting.Other
 {
     [RyuJitX64Job]
-    public class CountBench : PerformanceTest
+    public class BenchCountBench : PerformanceTest
```

Comment:
After several name changes. I think that it should be different prs next time. One for "Reverse matrix" and one for "Perf tests names change".